code/modules/clothing/neck/mantle.dm

```diff
@@ -0,0 +1,135 @@
+
+/obj/item/clothing/neck/mantle/ranger
+	name = "NCR ranger cape"
+	desc = "Ranger cape made from what looks like old poncho fitted with star, stripes and a bear. Most likely has a holster hidden underneath."
+	icon_state = "rangercape"
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/bos/paladin
+
+/obj/item/clothing/neck/mantle/chief
+	name = "chieftains mantle"
+	desc = "A symbol of the authority of the Wayfarer Chief."
+	icon_state = "tribechief"
+
+/obj/item/clothing/neck/mantle/jay
+	name = "blue silk mantle"
+	desc = "A finely woven and blue dyed mantle, with the emblem of a bird on its back."
+	icon_state = "jaymantle"
+
+/obj/item/clothing/neck/mantle/enclavecommander
+	name = "commanders mantle"
+	desc = "A fine mantle marking the wearer as a Commander of the Enclave."
+	icon_state = "commander"
+
+/obj/item/clothing/neck/mantle/treasurer
+	name = "treasurers mantle"
+	desc = "The grey and black mantle with gold thread trimming shows the wearer is entrusted with matters of money and records. Hidden inner pockets can store money, keys and documents safely, and a discrete sheath for a knife for self defence is also attached."
+	icon_state = "treasurer"
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/treasurer
+
+/obj/item/clothing/neck/mantle/peltfur
+	name = "fur pelt"
+	desc = "A pelt made from longhorner fur."
+	icon_state = "peltfur"
+	cold_protection = CHEST|GROIN|ARMS
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+
+/obj/item/clothing/neck/mantle/peltmountain
+	name = "fur pelt"
+	desc = "A pelt made from a mountain bear, brought in from Colorado."
+	icon_state = "peltmountain"
+	cold_protection = CHEST|GROIN|ARMS
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+
+/obj/item/clothing/neck/mantle/ragged
+	name = "ragged mantle"
+	desc = "A worn and ripped old primitive mantle, made from sinew and bone."
+	icon_state = "ragged"
+
+/obj/item/clothing/neck/mantle/poncho
+	name = "poncho"
+	desc = "Plain and rugged piece of clothing, to make sure your gear don't get soaked through when it rains."
+	icon_state = "poncho"
+	body_parts_covered = CHEST|GROIN|ARMS
+
+/*
+Obsolete
+/obj/item/clothing/neck/cloak
+
+/obj/item/clothing/suit/poncho
+	name = "poncho"
+	desc = "(I) Your classic, non-racist poncho."
+	icon_state = "classicponcho"
+	item_state = "classicponcho"
+	permeability_coefficient = 0.8
+
+/obj/item/clothing/suit/poncho/green
+	name = "green poncho"
+	desc = "(I) Your classic, non-racist poncho. This one is green."
+	icon_state = "greenponcho"
+	item_state = "greenponcho"
+
+/obj/item/clothing/suit/poncho/red
+	name = "red poncho"
+	desc = "(I) Your classic, non-racist poncho. This one is red."
+	icon_state = "redponcho"
+	item_state = "redponcho"
+*/
```

Comment:
```suggestion
```

Reply:
Need it to remain for now as a reminder for myself where to start the cleanup PR, as discussed.